ember-toucan-core/src/components/form/file-input-field.hbs

```diff
@@ -0,0 +1,39 @@
+<div
+  class="flex flex-col {{if @isDisabled 'text-disabled'}}"
+  data-root-field={{if @rootTestSelector @rootTestSelector}}
+>
+  <Form::Field as |field|>
+    <field.Label for={{field.id}}>
+      <span data-label>{{@label}}</span>
+      {{#if @hint}}
+        <field.Hint data-hint>{{@hint}}</field.Hint>
+      {{/if}}
+
+      <field.Control class="mt-1.5">
+        <Form::Controls::FileInput
+          aria-describedby="{{if this.hasError field.errorId}}"
+          aria-invalid={{if this.hasError "true"}}
+          @hasError={{this.hasError}}
+          accept={{this.accept}}
+          multiple={{this.multiple}}
+          id={{field.id}}
+          @onChange={{@onChange}}
```

Comment:
Shouldn't this be `@onChange={{this.onChange}}`? Otherwise `this.onChange` would be unused.

Reply:
Okay this was a mistake on my part, but a bit messy so I am going to break down what I did (for my benefit, mostly)

1. in the controls/file-input, there is an `on` handler which takes a `this.onChange` handler. That handler only takes an `event` as a parameter, because it's using `on`.
2. So in `controls/file-input`, in the handler, I then call the `onChange` with the files array, AND the event. 
3. in `file-input-field` this should actually YES, be `@onChange={{@onChange}}`. What was wrong is I forgot to delete the `this.onChange` from the backing class.